src/actions/sso/authenticationRequest.ts

```diff
@@ -21,10 +23,16 @@ export const consumeAuthenticationRequest = (
   authenticationRequest: JSONWebToken<Authentication>,
   isDeepLinkInteraction: boolean = false,
 ): ThunkAction => async (dispatch, getState, backendMiddleware) => {
-  const { identityWallet } = backendMiddleware
+  const { identityWallet, registry } = backendMiddleware
   await identityWallet.validateJWT(authenticationRequest)
+  const requester = await registry.resolve(
+    keyIdToDid(authenticationRequest.issuer),
+  )
+
+  const requesterSummary = parsePublicProfile(requester)
```

Comment:
Should this be renamed to `getRequestSummary` or `toIdentitySummary`? Not 100% sure myself.

Reply:
Renamed it to `getidentitySummary`